google/cloud/storage/internal/async/reader_connection_tracing.cc

```diff
@@ -13,10 +13,11 @@
 // limitations under the License.
 
 #include "google/cloud/storage/internal/async/reader_connection_tracing.h"
```

Comment:
inconsequential comment: this include could go in the `#if` block too.

Reply:
I would say it is best to keep it as is, as the ability to make the inclusion is not a conditional thing, and should be exercised.  It is only the ability to do anything after the inclusion that is conditional.